test/test_dist_autograd_fork.py

```diff
@@ -1,21 +1,23 @@
 #!/usr/bin/env python3
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import process_group_dist_autograd_test
 import torch
-
-# dist_autograd_fork tests use double as the default dtype
-torch.set_default_dtype(torch.double)
-
-from dist_autograd_test import DistAutogradTest
 from common_distributed import MultiProcessTestCase
 from common_utils import run_tests
 
 
-class DistAutogradTestWithFork(MultiProcessTestCase, DistAutogradTest):
+# dist_autograd_fork tests use double as the default dtype
+torch.set_default_dtype(torch.double)
```

Comment:
Why float32 is not enough?

Reply:
I don't know the reason. I triggered auto formatter. Reverting these few lines.